rails_event_store_active_record/Gemfile.rails_6_0.lock

```diff
@@ -0,0 +1,241 @@
+PATH
```

Comment:
Instance of: https://github.com/RailsEventStore/rails_event_store/pull/905/files#r542507198